source/agora/consensus/data/genesis/Test.d

```diff
@@ -59,44 +59,44 @@ public immutable Block GenesisBlock = {
             // NODE6
```

Comment:
I think these node comments will now need updating also. When https://github.com/bpfkorea/agora/pull/1626 is merged it will be easier as they get printed when the enrolments need updating.

Reply:
Updated. Thanks.